examples/csv/src/main.rs

```diff
@@ -31,7 +31,10 @@ fn main() -> Result<(), std::io::Error> {
 
     let mut stdout = std::io::stdout();
 
-    match Dump::new(&args.file, args.addresses) {
+    match Dump::new(
+        &args.file,
+        ComputeAddresses::Yes(txoutset::Network::Bitcoin),
```

Comment:
Does this ignore the `addresses` argument?

Reply:
Indeed. Addressed.